enhancements/update/overriding-blocked-edges/README.md

```diff
@@ -0,0 +1,58 @@
+# Overriding Blocked Edges
+
+We work hard to ensure update and release stability before even cutting candidate release images, but sometimes bugs slip through, as you can see [by our blocked edges][blocked-edges].
+This can make users uncomfortable.
+For example, a user might:
+
+1. See a new A -> B edge appear in their channel.
+2. Begin sending stage clusters over the edge to test its stability for their cluster flavor.
+3. Become confident in the update's stability for their flavor.
+4. Schedule a maintenance window to update their production cluster(s).
+5. Have the edge pulled from their channel, causing distress when the maintenance window arrives and the A -> B edge is no longer recommended.
+
+There have been many requests about rendering update graphs or pushing notifications about edge-pulls to mitigate these concerns, but I'm still not clear on how useful those would be.
+I understand the user's decision tree to flow something like:
+
+<div style="text-align:center">
+  <img src="flow.svg" width="100%" />
+</div>
+
+I've marked the risky flows dashed and orange and the conservative flows thick and blue.
+
+For me the sticking point is: who would ever say "yes" to "Are we confident Red Hat's reasoning is exhaustive?"?
+If you feel like there are bugs you missed in your local testing but that there are no bugs which Red Hat has missed, why are you even doing local testing?
+I've marked the "What bug(s) did Red Hat link?" node dashed and orange to show that I consider it a risky pathway that we do not need to enable with a convenient, structured response.
```

Comment:
Assuming customers look in to cincinnati-graph-data repository to see what BZ caused the edge removal is not right. Currently we have not gone down the path for customers i.e. not communicated about cincinnati-graph-data to customers explicitly AFAIK

Reply:
> Assuming customers look in to cincinnati-graph-data repository to see what BZ caused the edge removal is not right. Currently we have not gone down the path for customers i.e. not communicated about cincinnati-graph-data to customers explicitly AFAIK

I am not recommending customers look at cincinnati-graph-data.  I am pointing out that knowing which bugs RH considered when blocking an edge doesn't seem like particularly useful information.  If you buy that argument, do we care about if/how users can discover the list of bugs?  I am happy they are public, but also happy they are in free-form comments and not exposed via a structured API.  I would not be too put out if they were private.  I am against exposing them in a public, structured API because that gives the impression that folks should care about them, and I don't see a workflow where that would make sense.